README.md

```diff
@@ -123,6 +123,29 @@ The process to add a new column from the ``TraceGC`` event is the following:
+
+(All color values can be defined using [Spectre Color strings](https://spectreconsole.net/appendix/colors) or hex values.)
+
+| Name                  | Description                                                                                                                                       |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| use_plain_text        | Set to true to force the output to be plain text. Default is false. Note that the [NO_COLOR initiative](https://no-color.org/) is also supported. |
+| gc_table_header_color | The color for the GC table headers.                                                                                                               |
+| gen0_row_color        | The color for row entries of Gen 0 events in the GC table.                                                                                        |
+| gen1_row_color        | The color for row entries of Gen 1 events in the GC table.                                                                                        |
+| gen2_row_color        | The color for row entries of Gen 2 events in the GC table.                                                                                        |
+| gen0_heap_color       | The color for gen0 stats in the heap stats table.                                                                                                 |
+| gen1_heap_color       | The color for gen1 stats in the heap stats table.                                                                                                 |
+| gen2_heap_color       | The color for gen2 stats in the heap stats table.                                                                                                 |
+| gen3_heap_color       | The color for gen3 (LOH) stats in the heap stats table.                                                                                           |
+| gen4_heap_color       | The color for gen4 (pinned object heap) stats in the heap stats table.                                                                            |
+| total_heap_color      | The color used to for information about total heap stats in the heap stats table.                                                                 |
+| highlight_color       | The color used to call out special information, like the process being monitored.                                                                 |
+| message_color         | The color used for standard messages written to output.                                                                                           |
+| warning_color         | The color used for warning messages written to output.                                                                                            |
```

Comment:
these seem to all be for the stats table? what about colors for printing out the individual GC events?

Reply:
The individual GC events are the "row_color" fields above this on lines 136-138.

Message/Warning color were not intended to be specific to the heap stats table but... I guess it ended up that way 😅The two other messages we write to output right now are "ruled" lines `-------- Message --------` which are currently covered by `MessageRuleColor` (which I forgot to document here!)

I think `MessageRuleColor` should be removed and we just use `MessageColor` for all general messages. The only warning right now is when there are no stats to print, but I think I'd lean towards keeping it as a general warning color config (not heap stats specific) for when other warnings are added. Let me know what you think!

Reply:
yes, I'm for keeping MessageColor for all general messages.